frontend/app/views/event/page.scala.html

```diff
@@ -49,7 +49,12 @@ <h1 class="event-masthead__name qa-event-detail-name">@event.name.text</h1>
                     }
                 </div>
             }
-            <img class="event-header__logo" src="@Asset.at("images/providers/" + event.providerOpt.getOrElse(event.metadata.identifier) + ".svg")">
+            @if(event.providerOpt.getOrElse(event.metadata.identifier) == "5x15") {
```

Comment:
Yeah, only thing I could think of would be returning a tuple of the name + extension, rather than going full case class.